packages/datadog_flutter_plugin/lib/src/rum/ddrum_method_channel.dart

```diff
@@ -26,7 +26,9 @@ class DdRumMethodChannel extends DdRumPlatform {
       internalLogger: core.internalLogger,
     );
 
-    methodChannel.setMethodCallHandler(callbackHandler.handleMethodCall);
+    if (ServicesBinding.rootIsolateToken != null) {
```

Comment:
This potentially could cause issues when using multiple flutters - but I understand this is not fully supported. 

I have zero clue if it actually will impact multiple flutters - I kinda think it WON'T simply because using two FlutterEngines here means we're not in a spawned isolate ever - but just wanted to mention it.

It might be worthwhile, if DD wants to support multiple flutter engines directly, to add a test / example based off of https://github.com/flutter/samples/tree/main/add_to_app/multiple_flutters. 

Appreciate the effort here regardless! 🎉

Reply:
Thanks for the input. I'll see if I can run a quick test with multiple flutters, but my guess there is that each engine will have its own root Isolate and therefore its own `rootIsolateToken`.